### DISCUSSION TITLE
Add "Duplicate" action button to browse page?

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3089] [Feature Requests]
Could be a right-click, or a new action button on selection.

This is what _other platforms_ do:

* Set status (which we don't have any more) to draft for new item
* Add a ` (1)` incremented value to the "main" field

This is very opinionated, so I don't think we can do either of these.

**Comment by rijkvanzanten** [comment]
Would this be batch-able? We already have the "Save as Copy" action on the detail page 🤔

**Reply by benhaynes** [reply]
Not sure — batch doesn't make much sense in this case. Yeah, we have this on the detail page, which might be enough/better... just throwing this out there as an option. Not sure if I even like the idea though.

**Reply by ghost** [reply]
A use case would be to duplicate lot’s of entries for testing in development …

**Reply by fitu86** [reply]
Esto se resuelve con flows desde la app.

![Screenshot_20230120_211308](https://user-images.githubusercontent.com/16139127/213841255-0f2b0ecc-e7b7-4475-baf3-d00786b5943c.jpg)

**Reply by ghost** [reply]
@fitu86 That is great! gracias

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
